website/docs/r/vapp_nat_rules.html.markdown

```diff
@@ -84,6 +82,14 @@ resource "vcd_vapp_nat_rules" "vapp-nat2" {
     vm_id        = vcd_vapp_vm.vm2.id
   }
 }
+
+# Must be defined just to enable FW for NAT rules to work if it wasn't enabled.
```

Comment:
1. Please move this resource above this line, to show the logical order:

`resource "vcd_vapp_nat_rules" "vapp-nat2" {`

2. Suggesting a more simple comment:

```suggestion
# In order to enable NAT service, firewall needs to be enabled as well
```

3. We also need a second FW resource above this line for the other network:

`resource "vcd_vapp_nat_rules" "vapp-nat" {`

Reply:
done

Reply:
Thanks! There's an extra line below the vapp_fw1 resource.

Reply:
removed